docs/website/docs/plus/ecosystem/snowflake_plus.md

```diff
@@ -124,6 +124,56 @@ Snowflake Plus supports all the data types from the standard Snowflake destinati
 | `binary` | `binary` |
 | `json` | `string` |
 
+
+## Syncing Snowflake-managed Iceberg tables to Snowflake Open Catalog
+
+To enable querying of Snowflake-managed Iceberg tables by third-party engines (e.g., Apache Spark) via an external catalog (Snowflake Open Catalog), use the `catalog_sync` configuration option. This setting specifies a [catalog integration](https://docs.snowflake.com/en/user-guide/tables-iceberg#catalog-integration) that syncs Iceberg tables to the external catalog.
+
+### Setup
+
+1. [Create an external catalog in Snowflake Open Catalog](https://other-docs.snowflake.com/en/opencatalog/create-catalog).
+
+2. Create a catalog integration in Snowflake. Example:
+
+```sql
+  CREATE OR REPLACE CATALOG INTEGRATION my_open_catalog_int
+    CATALOG_SOURCE = POLARIS
+    TABLE_FORMAT = ICEBERG
+    REST_CONFIG = (
+      CATALOG_URI = 'https://<orgname>-<my-snowflake-open-catalog-account-name>.snowflakecomputing.com/polaris/api/catalog'
+      CATALOG_NAME = 'myOpenCatalogExternalCatalogName'
+    )
+    REST_AUTHENTICATION = (
+      TYPE = OAUTH
+      OAUTH_CLIENT_ID = 'myClientId'
+      OAUTH_CLIENT_SECRET = 'myClientSecret'
+      OAUTH_ALLOWED_SCOPES = ('PRINCIPAL_ROLE:ALL')
+    )
+    ENABLED = TRUE;
+```
+
+Refer to the [Snowflake documentation](https://docs.snowflake.com/en/user-guide/tables-iceberg-open-catalog-sync#step-4-create-a-catalog-integration-for-open-catalog) for detailed setup instructions.
+
+3. Configure the `catalog_sync` option in your `config.toml` or `dlt.yml` file:
+
+- In `config.toml`:
```

Comment:
Probably makes sense to use one of those tabbed components here

Reply:
Yes, good point.

Reply:
Added in 331ca45